### PR TITLE
SLB-269: finish multilingual features

### PR DESCRIPTION
**Pull request by pmelab**
## Description of changes

* Make content hub react to the current language
* Add a simple language switcher
* Add interface translations to decap
* Add interface translations to drupal
* Integration tests for all of these

**Note:** This contains copies for files from https://github.com/AmazeeLabs/silverback-mono/pull/1494. As soon as it is merged, they will be removed from here.

## Motivation and context

Make the starter kit fully multilingual. With Drupal as well as Decap

## How has this been tested?

- [x] Manually
- [x] Unit tests
- [x] Integration tests

### CODE REVIEW

File: apps/cms/prep-database.sh [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-then
+if ! test -f web/sites/default/files/.sqlite; then
   pnpm drupal-install
   pnpm export-webforms
-else
-  pnpm drush cr
 fi
+
+# In any case, re-import translation string.
+pnpm import-translations
```
Comment (Leksat) [inline apps/cms/prep-database.sh:12]: Should we import translations during the Lagoon deployments?

**Review by Leksat — COMMENTED** [review]
I think the UI Test failure is a real one :)

File: apps/cms/prep-database.sh (hunk 1 above)
Comment (pmelab) [reply apps/cms/prep-database.sh:12]: I think so? Unless i'm misunderstanding something 😬 
When frontend code changes, there might be new translations and we have to import them.

Comment (Leksat) [reply apps/cms/prep-database.sh:12]: I mean, this file isn't executed in Lagoon context. And I haven't seen `import-translations` anywhere else in the codebase. So perhaps we need to add something to Dockerfile?

Comment (pmelab) [reply apps/cms/prep-database.sh:12]: Ah, no. This has already been there before. Its in a post-deploy step: https://github.com/AmazeeLabs/silverback-template/blob/b5c153a384218b611d112b3eef69d93d893b8afb/.lagoon.yml#L22

Comment (Leksat) [reply apps/cms/prep-database.sh:12]: Ah!

**Comment by pmelab** [comment]
> I think the UI Test failure is a real one :)

It is not 😛

**Review by Leksat — APPROVED** [review]

**Pull request merged**